app/components/primer/alpha/dialog.pcss

```diff
@@ -21,6 +21,15 @@ dialog.Overlay:not([open]) {
   opacity: 0;
 }
 
+/* hacks until firefox fully supports popovers */
+:where([popover]:not(.\:popover-open)) {
+  display: none !important;
+}
+
+:popover-open {
+  display: inherit !important;
+}
+
```

Comment:
This seems a bit globally dangerous, will it effect current popovers on the sight?

Reply:
I'm not sure 🤔 If it does, it should only affect Firefox users. It seems to work ok in Lookbook, but some careful testing will need to be done in dotcom. I will integrate and test before merging this PR.

Reply:
Integration testing PR: https://github.com/github/github/pull/315906